pkg/clustermanager/cluster.go

```diff
@@ -338,7 +350,10 @@ func (manager *Manager) InstallWorkers(nodes []Node) error {
 			numProcs++
 			go func(node Node) {
 				manager.eventService.AddEvent(node.Name, "registering node")
-				_, err := manager.nodeCommunicator.RunCmd(node, "kubeadm reset && "+joinCommand)
+				_, err := manager.nodeCommunicator.RunCmd(
+					node,
+					"for i in ip_vs ip_vs_rr ip_vs_wrr ip_vs_sh nf_conntrack_ipv4; do modprobe $i; done"+
```

Comment:
This could be the main issue why kubeadm join command was failing which is related to https://github.com/kubernetes/kubeadm/issues/975

So enabling those modules fixed the warning.